module/VuFindConsole/src/VuFindConsole/Command/Util/SwitchDbHashCommand.php

```diff
@@ -176,6 +145,28 @@ protected function fixRow($row, ?BlockCipher $oldcipher, BlockCipher $newcipher)
         $row->save();
     }
 
+    /**
+     * Re-encrypt an entity.
+     *
+     * @param AbstractDbService       $service   Database service
+     * @param UserCardEntityInterface $entity    Row to update
+     * @param ?BlockCipher            $oldcipher Old cipher (null for none)
+     * @param BlockCipher             $newcipher New cipher
+     *
+     * @return void
+     * @throws InvalidArgumentException
+     */
+    protected function fixEntity($service, $entity, ?BlockCipher $oldcipher, BlockCipher $newcipher): void
```

Comment:
Could types for `$service` and `$entity` be added? Also why `AbstractDbService`?

Reply:
You're right, `AbstractDbService` should be `DbServiceInterface`. I didn't add a type to `$entity` because I'm going to have to change the type when I add support for user entities in a future commit, but I guess it doesn't hurt to just set it up now and finish implementing it later.